apps/framework-docs/src/pages/building/make-changes.mdx

```diff
@@ -1,20 +1,25 @@
 # Changing your data models
-Change management in your data stack is critical to ensure that your data is always accurate and up to date based on your end user's expectations. Bad change management can mean broken dashboards, reports, or machine learning models can lead to unhappy customers and lost revenue. MooseJS makes it easy to change your data models and ensure that your data stack is always up to date. To do so, we leverage git workflows to ensure that changes are tracked and can be rolled back if necessary.
+
+Change management in your data stack is critical to ensure that your data is always accurate and up to date based on your end user's expectations. Bad change management can lead to broken dashboards, reports, or machine learning models, which can lead to unhappy customers and lost revenue. MooseJS makes it easy to change your data models and ensure that your data stack is always up to date. To do so, we leverage git workflows to ensure that changes are tracked and can be rolled back if necessary.
 
 ## Adding a new data model
-Adding a new data source is as simple as adding a new model to an existing or new file in the `data models` directory. MooseJS will automatically detect the new model and add it to the data stack. 
+
+Adding a new data source is as simple as adding a new model to an existing or new file in the `datamodels` directory. MooseJS will automatically detect the new model and add it to the data stack.
```

Comment:
Here I removed the space between the words `data` and `models` because that matches our directory structure.